jenkins/check-for-build.jenkinsfile

```diff
@@ -66,9 +71,17 @@ pipeline {
                             echo "Skipping, ${sha.path} already exists."
                         } else {
                             try {
-                                build job: "${TARGET_JOB_NAME}", parameters: [
-                                string(name: 'INPUT_MANIFEST', value: "${INPUT_MANIFEST}")
-                                ], wait: true
+                                // Currently only opensearch build takes an additional test manifest parameter
+                                if (TARGET_JOB_NAME == 'distribution-build-opensearch') {
+                                    build job: "${TARGET_JOB_NAME}", parameters: [
+                                    string(name: 'INPUT_MANIFEST', value: "${INPUT_MANIFEST}"),
+                                    string(name: 'TEST_MANIFEST', value: "${TEST_MANIFEST}")
+                                    ], wait: true
+                                } else if (TARGET_JOB_NAME == 'distribution-build-opensearch-dashboards') {
+                                    build job: "${TARGET_JOB_NAME}", parameters: [
+                                    string(name: 'INPUT_MANIFEST', value: "${INPUT_MANIFEST}")
+                                    ], wait: true
+                                }
```

Comment:
This is a hack, but at least it should have `else error`, otherwise if this is ever passed to a job that's not distribution-build-... there's no failure and nothing is kicked off.

Then, does passing TEST_MANIFEST into `distribution-build-opensearch-dashboards` that doesn't expect it actually fails?

Reply:
It's NBD, but maybe a comment would suffice.